frontend/src/App.js

```diff
@@ -5,6 +5,8 @@ import { Global, css } from '@emotion/core'
 import { PageNotFound } from './PageNotFound'
 import { LandingPage } from './LandingPage'
 import { DomainsPage } from './DomainsPage'
+import {SignInPage} from "./components/SignInPage/SignInPage"
```

Comment:
If we could get rid of the components directory that would be good. Essentially, everything is a component. I think we should stick with dogpiling them into the src directory for now and once we get this a little further advanced, and some good component boundaries we are happy with we can look to extract things into libraries.